examples/ecs/aws-cloudwatch/ecs-ec2-instance-daemon.json

```diff
@@ -0,0 +1,119 @@
+{
+  "executionRoleArn": "{{ecsTaskExecutionRoleArn}}",
+  "containerDefinitions": [
+    {
+      "dnsSearchDomains": [],
+      "environmentFiles": [],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "secretOptions": [],
+        "options": {
+          "awslogs-group": "/ecs/ecs-cwagent-daemon-service",
+          "awslogs-region": "us-west-2",
+          "awslogs-create-group": "True",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "command": [
+        {{command}}
+      ],
+      "linuxParameters": null,
+      "cpu": 0,
+      "environment": [
+        {
+          "name": "USE_DEFAULT_CONFIG",
+          "value": "True"
+        }
+      ],
+      "resourceRequirements": null,
+      "ulimits": [],
+      "dnsServers": [],
+      "mountPoints": [
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/proc",
+          "sourceVolume": "proc"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/dev",
+          "sourceVolume": "dev"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/sys/fs/cgroup",
+          "sourceVolume": "al2_cgroup"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/cgroup",
+          "sourceVolume": "al1_cgroup"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/sys/fs/cgroup",
+          "sourceVolume": "al2_cgroup"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/cgroup",
+          "sourceVolume": "al1_cgroup"
+        }
+      ],
+      "image": "amazon/aws-otel-collector",
+      "essential": true,
+      "name": "aws-collector"
+    }
+  ],
+  "memory": "64",
+  "taskRoleArn": "{{ecsTaskRoleArn}}",
+  "family": "ecs-otel-daemon-service",
+  "pidMode": null,
+  "requiresCompatibilities": [
+    "EC2"
+  ],
+  "networkMode": "bridge",
+  "cpu": "128",
+  "inferenceAccelerators": [],
+  "proxyConfiguration": null,
+  "volumes": [
+    {
+      "fsxWindowsFileServerVolumeConfiguration": null,
+      "efsVolumeConfiguration": null,
+      "name": "proc",
+      "host": {
+        "sourcePath": "/proc"
+      },
+      "dockerVolumeConfiguration": null
+    },
+    {
+      "fsxWindowsFileServerVolumeConfiguration": null,
```

Comment:
I think you can skipp those `null` fields.

Reply:
Sure, will delete them